src/main/java/com/searchservice/app/rest/InputDocumentResource.java

```diff
@@ -34,156 +28,117 @@
-		LoggersDTO loggersDTO = LoggerUtils.getRequestLoggingInfo(servicename, username, nameofCurrMethod, timestamp);
-		LoggerUtils.printlogger(loggersDTO, true, false);
-		loggersDTO.setCorrelationid(loggersDTO.getCorrelationid());
-		loggersDTO.setIpaddress(loggersDTO.getIpaddress());
-
-        // Apply RequestSizeLimiting Throttler on payload before service the request
+	@Operation(summary = "ADD DOCUMENTS IN THE TABLE OF THE GIVEN TENANT ID. INPUT SHOULD BE A LIST OF DOCUMENTS SATISFYING THE TABLE SCHEMA.", security = @SecurityRequirement(name = "bearerAuth"))
+	public ResponseEntity<ThrottlerResponse> document(@PathVariable int tenantId, @PathVariable String tableName,
+			@RequestBody String payload) {
+
+		if (!inputDocumentService.isValidJsonArray(payload))
+			throw new InvalidJsonInputOccurredException(HttpStatusCode.INVALID_JSON_INPUT.getCode(),
+					HttpStatusCode.INVALID_JSON_INPUT.getMessage());
+
+		// Apply RequestSizeLimiting Throttler on payload before service the request
 		ThrottlerResponse documentInjectionThrottlerResponse = throttlerServicePort
 				.documentInjectionRequestSizeLimiter(payload, false);
 		if (documentInjectionThrottlerResponse.getStatusCode() == 406)
 			return ResponseEntity.status(HttpStatus.NOT_ACCEPTABLE).body(documentInjectionThrottlerResponse);
 
 		// Control will reach here ONLY IF REQUESTBODY SIZE IS UNDER THE SPECIFIED LIMIT
 
-		tableName = tableName+"_"+tenantId;
-		 if(documentInjectionThrottlerResponse.getStatusCode() == 406)
-	        	return ResponseEntity
-	        			.status(HttpStatus.NOT_ACCEPTABLE)
-	        			.body(documentInjectionThrottlerResponse);
-
-	        // Control will reach here ONLY IF REQUESTBODY SIZE IS UNDER THE SPECIFIED LIMIT
-	      if(manageTableServicePort.isTableExists(tableName)) {
-	       	return performDocumentInjection(tableName,payload,documentInjectionThrottlerResponse,loggersDTO);
-	      }else {
-	         return documentInjectWithInvalidTableName(tenantId, tableName.split("_")[0]);
-	      }
-    }
-
-
-    // Rate Limiter(Throttler) FALLBACK method
-	public ResponseEntity<ThrottlerResponse> documentInjectionRateLimiterFallback(
-			int tenantId,
-			String tableName,
-			String payload, 
-			RequestNotPermitted exception) {
+		tableName = tableName + "_" + tenantId;
+		if (documentInjectionThrottlerResponse.getStatusCode() == 406)
+			return ResponseEntity.status(HttpStatus.NOT_ACCEPTABLE).body(documentInjectionThrottlerResponse);
+
+		// Control will reach here ONLY IF REQUESTBODY SIZE IS UNDER THE SPECIFIED LIMIT
+		if (manageTableServicePort.isTableExists(tableName)) {
+			return performDocumentInjection(tableName, payload, documentInjectionThrottlerResponse);
+		} else {
+			return documentInjectWithInvalidTableName(tenantId, tableName.split("_")[0]);
+		}
+	}
+
+	// Rate Limiter(Throttler) FALLBACK method
+	public ResponseEntity<ThrottlerResponse> documentInjectionRateLimiterFallback(RequestNotPermitted exception) {
```

Comment:
Revert changes you made here.., ignore code smell related to that one.., otherwise Rate Limiter will not work